src/site/content/en/blog/web-otp/index.md

```diff
@@ -174,19 +174,16 @@ doesn't matter who the sender is when using the Web OTP API.
 1. Send your phone the following SMS text message from the another phone.
 
 ```text
+Your OTP is: 123456.
+
 @web-otp.glitch.me #12345
 ```
 
 Did you receive the SMS and see the prompt to enter the code to the input area?
 That is how the Web OTP API works for users.
 
 {% Aside 'caution' %}
-* If the sender's phone number is included in the receiver's contact list, this API
-  will not be triggered due to the design of the underlying [SMS
-User Consent  API](https://developers.google.com/identity/sms-retriever/user-consent/request#2_start_listening_for_incoming_messages).
-* If you are using a work profile on your Android device and the Web OTP does
-  not work, try installing and using Chrome on your personal profile instead
-  (i.e. the same profile in which you receive SMS messages).
+The dialog doesn't appear for you? Check out [the FAQ](#no-dialog).
```

Comment:
Why not just explain it here?

Reply:
```suggestion
If the dialog doesn't appear for you, check out [the FAQ](#no-dialog).
```

Reply:
I'm OK with how Eiji has handled it. Per https://kayce.basqu.es/blog/FAQs/ (which was inspired by you, Joe), the main thing is that Eiji has mentioned the topic in a contextually relevant place. I'm OK with us having an FAQ, so long as that isn't the only source of the information.